### PR TITLE
Update site links & replace old apps

### PR DESCRIPTION
**Pull request by DimitrisPa**
**Remove / Replace :**

- PulseEffects for Easy Effects _(This application was formerly known as PulseEffects, but it was renamed to Easy Effects after it started to use GTK4 and GStreamer usage was replaced by native PipeWire filters.)_
- Backgammon, Greed, Ninvaders, Tron _(No available links for the site or the source code)_
- RecApp for Kooha _(This project is archived, it's no longer under development. If you would like to continue development, please feel free to fork the project according to the GPL license.)_
- Whatever _(Electron app, last update Feb 10, 2017)_
- antimicro for AntiMicroX _(This repo is currently unmaintained. The code hasn't been updated for a while. There is new recommended version of this app called AntiMicroX)_

**Update links for :**

- GNOME Circle Apps
- Irssi
- Conky
- Polybar
- oh-my-zsh
- Sigil
- Komodo
- Google Earth
- KTouch
- Bastet
- menyoki
- Flameshot
- kaku
- albert
- qownnotes
- ProxyChains
- ops
- MEGA
- GNOME Connections
- Alacritty
- Kitty
- Zellij
- Stacer
- GNOME Pie
- Gramps
- sushi
- Mycroft
- Daily Reddit Wallpaper
- Kmon
- OneFetch
- PathPicker
- Silver Searcher

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -207,7 +206,7 @@
 - [![Open-Source Software][OSS Icon]](https://github.com/bit-team/backintime/) [Back In Time](https://github.com/bit-team/backintime/) - A simple backup tool for Linux, inspired by "flyback project".
 - [![Open-Source Software][OSS Icon]](https://borgbackup.readthedocs.io/en/stable/development.html) [BorgBackup](https://borgbackup.readthedocs.io/en/stable/) - A deduplicating backup program with compression and authenticated encryption.
 - [![Open-Source Software][OSS Icon]](https://github.com/bup/bup) [bup](https://bup.github.io/) - Very efficient backup system based on the git packfile format, providing fast incremental saves and global deduplication (among and within files, including virtual machine images).
-- [![Open-Source Software][OSS Icon]](https://gitlab.gnome.org/World/deja-dup) [Deja Dup](https://launchpad.net/deja-dup) - A simple backup tool with built-in encryption.
+- [![Open-Source Software][OSS Icon]](https://gitlab.gnome.org/World/deja-dup) [Deja Dup](https://wiki.gnome.org/Apps/DejaDup) - A simple backup tool with built-in encryption.
```
Comment (FabioLolix) [inline README.md:209]: What about using https://apps.gnome.org/app/org.gnome.DejaDup/ as link? I've seen you have used apps.gnome.org when applicable

File: README.md [hunk 2]
```diff
@@ -1109,12 +1102,11 @@ the WebKit rendering engine.
 - [Termius](https://www.termius.com/) - Cross-platform terminal with built-in SSH and Telnet.
 - [![Open-Source Software][OSS Icon]](https://github.com/lanoxx/tilda) [Tilda](https://github.com/lanoxx/tilda) - A Gtk based drop down terminal for Linux and Unix.
 - [![Open-Source Software][OSS Icon]](https://github.com/gnunn1/tilix) [Tilix](https://gnunn1.github.io/tilix-web/) - A tiling terminal emulator for Linux using GTK+ 3.
-- [![Open-Source Software][OSS Icon]](https://github.com/tmux/tmux) [Tmux](https://github.com/tmux/tmux) - It lets you switch easily between several programs in one terminal, detach them (they keep running in the background) and reattach them to a different terminal. And do a lot more.
```
Comment (FabioLolix) [inline README.md:1112]: I guess removing tmux was a typo ?!

btw, the github org have the link http://tmux.github.io/ which redirects to the github wiki page https://github.com/tmux/tmux/wiki

File: README.md (hunk 1 above)
Comment (DimitrisPa) [reply README.md:209]: They use https://wiki.gnome.org/Apps/DejaDup as their Homepage link, in the Gitlab repo.

File: README.md (hunk 2 above)
Comment (DimitrisPa) [reply README.md:1112]: Duplicate entry, tmux is also in the Tools sub-category.

File: README.md (hunk 1 above)
Comment (FabioLolix) [reply README.md:209]: ok

File: README.md (hunk 2 above)
Comment (FabioLolix) [reply README.md:1112]: ok

**Pull request merged**